### PR TITLE
New: Utility to run webhint in a web worker

### PR DESCRIPTION
**Pull request by antross**
## Pull request checklist

Make sure you:

- [X] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [X] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [X] Added/Updated related documentation.
- [X] Added/Updated related tests.

### CODE REVIEW

File: packages/utils-worker/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Web worker for webhint
+
+The webhint web worker utility provides a bundled version of webhint
+suitable for running inside a web worker in a browser environment.
+
+## Installation
+
+This package can be installed via NPM by running the following:
```
Comment (molant) [inline packages/utils-worker/README.md:8]:
```suggestion
This package can be installed via npm by running the following:
```

File: packages/utils-worker/package.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js",
+      "!dist/tests/**/fixtures/**/*.js",
+      "!dist/tests/helpers/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "bundleSize": 2500000,
```
Comment (molant) [inline packages/utils-worker/package.json:11]: 😵

File: packages/utils-worker/scripts/bundle-size.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
```
Comment (molant) [inline packages/utils-worker/scripts/bundle-size.js:1]: Aren't we duplicating this script in a few places? Do we have an issue to refactor and just have it in one place?

File: packages/utils-worker/src/shims/get-message.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+     * ../hint-axe/dist/src
+     */
+    const pathParts = path.split(/\\|\//g);
+    const packageName = pathParts[pathParts.length - 3];
+    const prefix = camelCase(packageName);
+    const substitutions = options && options.substitutions;
+    const name = `${prefix}_${key}`;
+
+    if (!(messages as any)[name]) {
+        console.log('Name', name);
```
Comment (molant) [inline packages/utils-worker/src/shims/get-message.ts:21]: Is this needed or just for debugging?

File: packages/utils-worker/package.json (hunk 2 above)
Comment (antross) [reply packages/utils-worker/package.json:11]: Yep, unlike `extension-browser` this one measures the bundle minified, but not gzipped. It's actually smaller than `extension-browser` in an apples-to-apples comparison, though :)

File: packages/utils-worker/src/shims/get-message.ts (hunk 4 above)
Comment (antross) [reply packages/utils-worker/src/shims/get-message.ts:21]: Just for debugging. I'll remove it.

File: packages/utils-worker/scripts/bundle-size.js (hunk 3 above)
Comment (antross) [reply packages/utils-worker/scripts/bundle-size.js:1]: Yep. We didn't have an issue for this, but now we do: https://github.com/webhintio/hint/issues/3652

**Pull request merged**